src/index.html

```diff
@@ -1,14 +1,88 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible"
+      content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+      href="./style.css">
+    <link rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+        <a class="moyo"
+          href="#home">
+          <img src="./images/logo.png" alt="MOYO-logo">
+        </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list--low">
+            <a class="is-active nav__link"
+              href="#apple"
+            >
```

Comment:
```suggestion
            <a
              class="is-active nav__link"
              href="#apple"
            >
```
Зазвичай прийнято переносити атрибути на нові рядки, якщо їх більше трьох . Але якщо вже переносиш, то необхідно враховувати відступи правильно